### PR TITLE
[PM 202] Activate Organization when Stripe Subscription is Activated

### PR DESCRIPTION
**Pull request by cyprain-okeke**
## Type of change

```
- [ ] Bug fix
- [X] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other
```

## Objective

Activate Organization when Stripe Subscription is Activated via the webhook

## Code changes

* **src/Billing/Controllers/StripeController.cs:** Check for Active subscription status and enable the organization if the subscription is active.

## Before you submit

- Please check for formatting errors (`dotnet format --verify-no-changes`) (required)
- If making database changes - make sure you also update Entity Framework queries and/or migrations
- Please add **unit tests** where it makes sense to do so (encouraged but not required)
- If this change requires a **documentation update** - notify the documentation team
- If this change has particular **deployment requirements** - notify the DevOps team

### CODE REVIEW

File: src/Billing/Controllers/StripeController.cs
Comment (r-tome) [inline src/Billing/Controllers/StripeController.cs:1]: This looks great but perhaps the code readability can be improved.
What do you think about creating variables:
```
var organizationId = ids.Item1.Value;
var userId = ids.Item2.Value;
```

Comment (eliykat) [reply src/Billing/Controllers/StripeController.cs:1]: [Named tuples](https://josef.codes/c-sharp-tips-and-tricks-02-named-tuples/) are great for this!

**Review by r-tome — CHANGES_REQUESTED** [review]
Declare the `organizationId` and `userId` variables up top and use them everywhere to replace `ids.Item1.Value` and `ids.Item2.Value`.

**Comment by cyprain-okeke** [comment]
> Declare the `organizationId` and `userId` variables up top and use them everywhere to replace `ids.Item1.Value` and `ids.Item2.Value`.

**Review by r-tome — APPROVED** [review]
Good job 👍

**Pull request merged**